pkg/domain/domain.go

```diff
@@ -2285,6 +2285,9 @@ func quitStatsOwner(do *Domain, mgr owner.Manager) {
 func (do *Domain) StartLoadStatsSubWorkers(ctxList []sessionctx.Context) {
 	statsHandle := do.StatsHandle()
 	for i, ctx := range ctxList {
+		// The sync load will affect how optimizer choose the plan.
+		// We need to assign high priority to it so that we can get the stats as quick as we can.
+		ctx.GetSessionVars().StmtCtx.Priority = mysql.HighPriority
```

Comment:
Should we reset the `StmtCtx.Priority` after it?

Reply:
The worker will not do other jobs. So the priority doesn't need changing.